### ISSUE TITLE
[Question] Tenant Code set number that will AbstractShell$ExitCodeException

### ISSUE THREAD

**Issue by Guanpx** [apache/dolphinscheduler#5359]
**Which version of DolphinScheduler:**
 -[1.3.x]

**Additional context**
Tenant Manage > Tenant Code > new Tenant that code was 1
When that Tenant run task , Exception : 

```
[INFO] 2021-04-22 20:25:53.876  - [taskAppId=TASK-3-11-12]:[115] - create linux os user : 1
[INFO] 2021-04-22 20:25:53.876  - [taskAppId=TASK-3-11-12]:[115] - execute cmd : sudo useradd -g dolphinscheduler
 1
[ERROR] 2021-04-22 20:25:53.949  - [taskAppId=TASK-3-11-12]:[105] - useradd: invalid user name '1'

org.apache.dolphinscheduler.common.shell.AbstractShell$ExitCodeException: useradd: invalid user name '1'

	at org.apache.dolphinscheduler.common.shell.AbstractShell.runCommand(AbstractShell.java:209)
	at org.apache.dolphinscheduler.common.shell.AbstractShell.run(AbstractShell.java:124)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execute(ShellExecutor.java:127)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:104)
	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:87)
	at org.apache.dolphinscheduler.common.utils.OSUtils.exeShell(OSUtils.java:395)
	at org.apache.dolphinscheduler.common.utils.OSUtils.exeCmd(OSUtils.java:385)
	at org.apache.dolphinscheduler.common.utils.OSUtils.createLinuxUser(OSUtils.java:290)
	at org.apache.dolphinscheduler.common.utils.OSUtils.createUser(OSUtils.java:264)
	at org.apache.dolphinscheduler.common.utils.FileUtils.createWorkDirAndUserIfAbsent(FileUtils.java:175)
	at org.apache.dolphinscheduler.server.worker.processor.TaskExecuteProcessor.process(TaskExecuteProcessor.java:139)
	at org.apache.dolphinscheduler.remote.handler.NettyServerHandler$1.run(NettyServerHandler.java:128)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Requirement or improvement**
- declaration Tenant CODE was not begin with number !!!!

**Comment by davidzollo** [comment]
> **Which version of DolphinScheduler:**
> -[1.3.x]
> 
> **Additional context**
> Tenant Manage > Tenant Code > new Tenant that code was 1
> When that Tenant run task , Exception :
> 
> ```
> [INFO] 2021-04-22 20:25:53.876  - [taskAppId=TASK-3-11-12]:[115] - create linux os user : 1
> [INFO] 2021-04-22 20:25:53.876  - [taskAppId=TASK-3-11-12]:[115] - execute cmd : sudo useradd -g dolphinscheduler
>  1
> [ERROR] 2021-04-22 20:25:53.949  - [taskAppId=TASK-3-11-12]:[105] - useradd: invalid user name '1'
> 
> org.apache.dolphinscheduler.common.shell.AbstractShell$ExitCodeException: useradd: invalid user name '1'
> 
> 	at org.apache.dolphinscheduler.common.shell.AbstractShell.runCommand(AbstractShell.java:209)
> 	at org.apache.dolphinscheduler.common.shell.AbstractShell.run(AbstractShell.java:124)
> 	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execute(ShellExecutor.java:127)
> 	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:104)
> 	at org.apache.dolphinscheduler.common.shell.ShellExecutor.execCommand(ShellExecutor.java:87)
> 	at org.apache.dolphinscheduler.common.utils.OSUtils.exeShell(OSUtils.java:395)
> 	at org.apache.dolphinscheduler.common.utils.OSUtils.exeCmd(OSUtils.java:385)
> 	at org.apache.dolphinscheduler.common.utils.OSUtils.createLinuxUser(OSUtils.java:290)
> 	at org.apache.dolphinscheduler.common.utils.OSUtils.createUser(OSUtils.java:264)
> 	at org.apache.dolphinscheduler.common.utils.FileUtils.createWorkDirAndUserIfAbsent(FileUtils.java:175)
> 	at org.apache.dolphinscheduler.server.worker.processor.TaskExecuteProcessor.process(TaskExecuteProcessor.java:139)
> 	at org.apache.dolphinscheduler.remote.handler.NettyServerHandler$1.run(NettyServerHandler.java:128)
> 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
> 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> 	at java.lang.Thread.run(Thread.java:748)
> ```
> 
> **Requirement or improvement**
> 
> * declaration Tenant CODE was not begin with number !!!!
good job!

Could you submit a PR to solve this issue? I think it's not difficult,  Please contact me(lidongdai@apache.org) if you have any question

if you want to solve this question , please leave message, I will assign this issue to you, thx

**Comment by chengshiwen** [comment]
@Guanpx Duplicated with issue #4383, and it has been solved by #5093

**Comment by Guanpx** [comment]
fine,  thx
